### PR TITLE
Add meetup topics and curriculum

### PR DESCRIPTION
**Pull request by girikuncoro**
This PR aims to be foundation of meetup topics and curriculum, suggested topics that meetup speakers can choose from. This topics are just meant for guideline, they don't necessarily need to follow the content and reference links. Speakers are given freedom to prepare their own materials, given topics that we suggest.

### CODE REVIEW

**Review by imrenagi — APPROVED** [review]

File: docs/MEETUP_TOPICS.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+11. **Intro to Helm**  
+Helm is a package manager for Kubernetes, like how `apt` is for `ubuntu`. This will not be in CKA exam but will be very useful, necessary, and practical to someone new to Kubernetes. Reference: [Helm docs](https://helm.sh/docs/intro/).
+
+12. **Extending Kubernetes**  
+This is the next step of Kubernetes, good for the last session of curriculum. It won't be in CKA exam but good for someone who has completed the entire core curriculum and looking to figure out what to learn next. This topic will cover [extending Kubernetes](https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster/) with [kubectl plugins](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/), [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/), what's [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) and [operator SDK](https://github.com/operator-framework/operator-sdk).
+
+
+#### Category #2: Coud Native Stack Exploration
```
Comment (2pai) [inline docs/MEETUP_TOPICS.md]: typo ? `Coud Native Stack Exploration` should be `Cloud Native Stack Exploration`

File: docs/MEETUP_TOPICS.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* [Envoy](https://www.envoyproxy.io/)
+* [Open Policy Agent](https://www.openpolicyagent.org/)
+* [CoreDNS](https://coredns.io/)
+* [gRPC](https://grpc.io/)
+
+## Intermediate Track
+
+### Objective
+
+This track will cover practical usecases of Kubernetes and cloud native technologies adopted by companies, or can cover any intro to CNCF projects (like in Category #2 of Beginner Track) if a meetup decides to pick Category #1 already in the Beginner Track.
```
Comment (2pai) [inline docs/MEETUP_TOPICS.md]: `use cases` instead of `usescases`

**Review by 2pai — CHANGES_REQUESTED** [review]

File: docs/MEETUP_TOPICS.md (hunk 1 above)
Comment (girikuncoro) [reply docs/MEETUP_TOPICS.md]: fixed in https://github.com/cloudnative-id/meetups/pull/33/commits/6a920f254edebd32fa1be3b8ec949a8015650052

File: docs/MEETUP_TOPICS.md (hunk 2 above)
Comment (girikuncoro) [reply docs/MEETUP_TOPICS.md]: fixed in https://github.com/cloudnative-id/meetups/pull/33/commits/6a920f254edebd32fa1be3b8ec949a8015650052

**Comment by joshuabezaleel** [comment]
How about we could adhere the 2nd category of Cloud Native Stack Exploration according to the CNCF TrailMap?

**Review by 2pai — APPROVED** [review]

**Comment by girikuncoro** [comment]
> How about we could adhere the 2nd category of Cloud Native Stack Exploration according to the CNCF TrailMap?

oh this is really nice idea! we can follow trailmap "curriculum", will revise this

**Comment by joshuabezaleel** [comment]
> > How about we could adhere the 2nd category of Cloud Native Stack Exploration according to the CNCF TrailMap?
> 
> oh this is really nice idea! we can follow trailmap "curriculum", will revise this

Yep, there are 10 section which we could divide it into 1 per month, and for a bigger section/topic we could use 2 months for that so that'll be 12. 
Hopefully by the end of the year attendees could already yield a "full cloud native" app.

**Review by prakashdivyy — APPROVED** [review]

**Review by zufardhiyaulhaq — APPROVED** [review]

**Pull request merged**